src/System.Net.NetworkInformation/tests/FunctionalTests/project.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "System.Net.Primitives": "4.0.10-beta-*",
+
+    "xunit": "2.1.0-beta3-*",
+    "xunit.netcore.extensions": "1.0.0-prerelease-*"
```

Comment:
There is some issue updating this to 
`    "xunit": "2.1.0",`
On my machine, I'm now getting: 
``` 
  Discovering: System.Net.NetworkInformation.Functional.Tests
  Discovered:  System.Net.NetworkInformation.Functional.Tests
  Info:        System.Net.NetworkInformation.Functional.Tests has no tests to run```

I'll rebase then build the full tree - maybe that fixes it.